content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,532 @@
+        "page": {
+            "after": "eyJhZnRlciI6IkFRQUFBWFVBWGQ5MU05d3lUZ0FBQUFCQldGVkJXR1E1TVZaclFtRnpkRVoyVEc5QlFRIn0"
+        }
+    },
+    "data": [
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "source:postman"
+                ],
+                "timestamp": "2020-10-07T00:01:41.909Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "700",
+                    "user": "steve",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXRYVJGxvDQAAAABBWFVBWFJZVmd2ZlktbUdUZjRBQQ"
+        },
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "source:postman"
+                ],
+                "timestamp": "2020-10-07T00:01:57.586Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "500",
+                    "user": "bob",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXVNSvuMvWwAAAABBWFVBWFZOU2I2ZWcxX3c2LVVBQQ"
+        },
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "source:postman"
+                ],
+                "timestamp": "2020-10-07T00:02:33.461Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "450",
+                    "user": "chris",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXd91M9wyTgAAAABBWFVBWGQ5MVZrQmFzdEZ2TG9BQQ"
+        }
+    ],
+    "links": {
+        "next": "https://api.datadoghq.com/api/v2/logs/events?sort=timestamp&filter%5Bquery%5D=%2A&page%5Bcursor%5D=eyJhZnRlciI6IkFRQUFBWFVBWGQ5MU05d3lUZ0FBQUFCQldGVkJXR1E1TVZaclFtRnpkRVoyVEc5QlFRIn0&filter%5Bfrom%5D=2020-10-07T00%3A00%3A00%2B00%3A00&filter%5Bto%5D=2020-10-07T00%3A15%3A00%2B00%3A00"
+    }
+}
+
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+### Limit the number of results retrieved
+
+With the following API call, limit the number of log events retrieved. The `limit` indicates the maximum number of log events returned in the response.
```

Comment:
You do want to indicate the maximum value there. Otherwise we might think it is infinite and therefore wonder why there is pagination.

Reply:
Especially as you say it in the next section :)